acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
 if sys.version_info < (2, 7):
     install_requires.extend([
         # only some distros recognize stdlib argparse as already satisfying
         'argparse',
-        'mock<1.1.0',
```

Comment:
I think we resolved this to our mutual satisfaction in IRC. Kuba's concern was that test_utils routines in acme might be called from future third-party acme clients and that test_utils-only dependencies would thus not get installed. We will solve this, if it arises, by keeping a [testing] extra in acme that includes mock and having the third-party client's `test_requires='acme[testing']`.